_drafts/conda.md

```diff
@@ -0,0 +1,698 @@
+> to uninstall Conda, deleting that directory will be all that is necessary.
+
+### Installing Conda
+
+To install the `conda` client on a machine, the recommended way is to download
+the interactive
+[Miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it
+through `bash`. The following steps apply to both Linux and macOS. If you are
+using Windows, there is a Miniconda installer that can be downloaded from the
+same page.
+
+```
+$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+$ bash Miniconda3-latest-MacOSX-x86_64.sh
+```
+
+The final step in the installer is to make sure the
+`$HOME/miniconda3/etc/profile.d/conda.sh` script is loaded in the shell. When
+the following prompt is given:
+
+```
+Do you wish the installer to initialize Miniconda3
+by running conda init? [yes|no]
+```
+
+I suggest saying `no` and modifying the appropriate shell startup file
+(`.bashrc` or `.zshrc`, etc.) to contain the following.
+
+```
+. "$HOME/miniconda3/etc/profile.d/conda.sh"
+```
+
+If everything was successful, after reloading our shell, we can now run the
+`conda` command in the shell.
+
+```
+$ conda --version
+conda 4.7.12
+```
+
+### Creating the Environment
+
+The next step is to create an isolated environment in which we can install
+packages into. We will create an environment called `example`.
+
+```
+$ conda create -n example
+Collecting package metadata (current_repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /Users/tyler/miniconda3/envs/example
+
+Proceed ([y]/n)? y
+
+Preparing transaction: done
+Verifying transaction: done
+Executing transaction: done
+#
+# To activate this environment, use
+#
+#     $ conda activate example
+#
+# To deactivate an active environment, use
+#
+#     $ conda deactivate
+
+```
+
+Now, we are able to load up our environment using:
+
+```
+$ conda activate example
+```
+
+Our current environment has zero packages installed. We can double check this by
+running `conda list`.
+
+```
+(example)
+$ conda list
+# packages in environment at /Users/tyler/miniconda3/envs/example:
+#
+# Name                    Version                   Build  Channel
+
+```
+
+### Configuring the Environment
+
+We want to install the following packages:
+
+- ARM GCC Toolchain (8-2019-q3-update)
+- Make (4.2.1)
+- Python (3.6)
+- nrfutil (6.0.0)
+
+There are two ways to install packages. The first is by installing them directly
+with the CLI (e.g. `conda install make`), and the second is by defining them in
+a `environment.yml` file, generally placed in the root of the project. We will
+use the second method.
+
+There are also multiple repositories to install packages from. The two most
+popular ones are "Anaconda" and "Conda Forge". We will use Conda Forge as our
+primary repository, since it has more mainstream packages that we as firmware
+developers use. We will also add my personal Conda repository (tyhoff17), as it
```

Comment:
Would be cool to have a Conda repo under Memfault rather than your personal one.

Reply:
Done!